### PR TITLE
Add Section Selector

### PR DESCRIPTION
**Pull request by PatrickCleary**
Add section selector.

### CODE REVIEW

File: components/Dropdowns/LineSelector.tsx [hunk 1]
```diff
@@ -34,34 +33,33 @@ const LineSelectionItem = ({ lineName, selectedLine }) => {
 
 export const LineSelector = ({ selectedLine }) => {
   return (
-    <Popover className="relative">
+    <Popover className="relative flex h-full items-center">
       {({ open }) => (
         <>
-          <Popover.Overlay className="fixed inset-0 bg-black opacity-30" />
+          <Popover.Overlay className="mb-safe fixed inset-0 bottom-11 bg-black opacity-30" />
```
Comment (PatrickCleary) [inline components/Dropdowns/LineSelector.tsx:39]: mb-safe is needed for chrome on mobile. When the bottom navigation collapses, it overlaps with the home slide-up thingy on iphone.

File: components/general/StatSelectionSlider.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
+.SSI-Invisible {
```
Comment (PatrickCleary) [inline components/general/StatSelectionSlider.css:1]: Lots of animation stuff that is kinda messy, but I think I'll rework this component entirely later on.

File: styles/globals.css [hunk 3]
```diff
@@ -39,3 +42,12 @@ a {
   width: 92%;
   max-width: 1200px;
 }
+
+.pb-safe {
```
Comment (PatrickCleary) [inline styles/globals.css:46]: This prevents the site from overlapping with the iphone home bar.

**Comment by skaplan-dev** [comment]
Could you seperate out the statistics changes and nav changes?

File: styles/globals.css [hunk 4]
```diff
@@ -4,11 +4,14 @@
 
 :root {
   --transitmatters-red: #d31a2b;
-  --red-line: #da291c;
+  --red-line: #d13434;
```
Comment (skaplan-dev) [inline styles/globals.css:7]: did we have the colors wrong here?

**Comment by PatrickCleary** [comment]
The red I have been using in the designs is the same red as the NTT, which
is different from the regular red. Now that you point this out though, I
probably shouldn’t have changed this value if it will affect the old
dashboards. I’ll fix this.

**Review by skaplan-dev — APPROVED** [review]
feel free to merge once we figure out the color stuff

**Pull request merged**